moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -59,6 +59,11 @@
 #include "ompl/base/objectives/StateCostIntegralObjective.h"
 #include "ompl/base/objectives/MaximizeMinClearanceObjective.h"
 
+namespace ompl_interface
+{
+constexpr char LOGNAME[] = "model_based_planning_context";
+}  // namespace ompl_interface
+
```

Comment:
[nit] Should this be an anonymous namespace? This is inside a cpp file so it won't be included anywhere.

It looks like this whole file could just be put in the `ompl_interface` namespace and then you'd be able to drop that namespace from before `ModelBasedPlanningContext` like on line 62.

Here is an example of what I'm suggesting: https://github.com/ros-planning/moveit/blob/7ad2bc7b86dad08061d98668ba34feba54bb05cc/moveit_ros/moveit_servo/src/low_pass_filter.cpp#L47

Reply:
> It looks like this whole file could just be put in the ompl_interface namespace

This is something I've been wanting to do for a while. It could also make some other files cleaner, but I thought there might be some guidelines against putting the whole source file in a namespace. (No idea why, just that I do not see another reason why these files are written this way.)

The anonymous namespace thing is interesting! It's a feels a bit counterintuitive that you can access variables inside an anonymous namespace :)